zh-CN/Adding_Glyphs_to_an_Arabic_Font.md

```diff
@@ -27,7 +27,7 @@ title: 添加字形到阿拉伯字体
 
 #### 为什么要重命名字体？
 
-如果你不重命名字体，你修改的字体将不会与原始的区分开安装&ndash;你将不得不首先卸载原始字体。如果你准备分发你的修改版的时候重命名字体也是明智的&ndash;如果字体原作者使用Reserved Font Name（RFN）机制保留了字体名，那么原始名字只能用于原作者的字体版本。
+如果你不重命名字体，你修改的字体将不会与原始的区分开安装&mdash;&mdash;你将不得不首先卸载原始字体。如果你准备分发你的修改版的时候重命名字体也是明智的&mdash;&mdash;如果字体原作者使用Reserved Font Name（RFN）机制保留了字体名，那么原始名字只能用于原作者的字体版本。
```

Comment:
Do the Chinese use double long dashes? In Latin typography at least, double *hyphens* are used where dashes can't be accessed (easily).